src/modules/rust.rs

```diff
@@ -321,6 +354,8 @@ mod tests {
         static OVERRIDES_CWD_A: &str = r"\\?\C:\src";
         static OVERRIDES_CWD_B: &str = r"C:\src";
 
+        fs::create_dir(OVERRIDES_CWD_B).unwrap();
```

Comment:
This test will pollute `C:\` and can only succeed once (dir already exists). Please add an `#[ignore]` to restrict it to CI-only or fix that.

Reply:
valid point, I'll change it.